<launch>
	<node name="dvl_raw" pkg="offboard_comms" exec="dvl_raw" output="screen" />
	<node name="dvl_odom" pkg="offboard_comms" exec="dvl_odom" output="screen" />
</launch>